scripts/hyperkit.js

```diff
@@ -0,0 +1,20 @@
+// This downloads hyperkit, builds it, and puts the binary in the right place.
+
+const fs = require('fs');
+const os = require('os');
+const { spawn, spawnSync } = require('child_process');
+
+fs.mkdirSync("./resources/" + os.platform(), { recursive: true });
+
+// The version of hyperkit to build
+let ver = "v0.20210107";
+
+// TODO: rewrite to remove callbackitis
```

Comment:
And also check the exit code (that can be in the follow-up too).

Reply:
Good idea. done.